.github/workflows/release.yml

```
@@ -8,6 +8,12 @@ on:
     paths:
       - 'releases/**'
 
+permissions:
+  contents: read
```

```suggestion
  contents: write
```

Better make it `write` as it sometimes will create commits on its own repo.